### PR TITLE
Items List Components and rendering data from a Custom Hook

### PR DESCRIPTION
**Pull request by LilianaRestrepoTorres**
Hello team! 🏂

This PR solves issues #10 and #11 .
Both components are created. Getting real data from a Custom Hook, then setting it into a State and using it from ItemsList component to render it.

- [x] Create components
- [x] Render data from an api
- [x] Responsive
- [x] Test

Desktop view:

Screen Shot 2022-06-15 at 10 40 29 PM

Mobile view:

Screen Shot 2022-06-15 at 10 40 45 PM

### CODE REVIEW

**Review by Hai5edfm — APPROVED** [review]
nice job 🦉

File: src/components/ListItems/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import { getModules } from "@customTypes/createItemTypes";
+import CardItem from "@components/CardItem";
+import {useGetModules} from '../../hooks/useGetModules'
+
+// It shoul be replaced when we have db connection
+const itemsData = [
+  {
```
[the remark this answers is not in the document]
Comment (LilianaRestrepoTorres) [reply src/components/ListItems/index.tsx:8]: Ohh yeah! I'll do it. Thanks.

File: src/components/ListItems/index.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    description:
+      "Lorem ipsum, dolor sit amet consectetur adipisicing elit. Maiores impedit perferendis suscipit eaque, iste dolor cupiditate blanditiis ratione.",
+  },
+  {
+    name: "Item 5",
+    description:
+      "Lorem ipsum, dolor sit amet consectetur adipisicing elit. Maiores impedit perferendis suscipit eaque, iste dolor cupiditate blanditiis ratione.",
+  },
+];
+const title = "Show Module"
```
[the remark this answers is not in the document]
Comment (LilianaRestrepoTorres) [reply src/components/ListItems/index.tsx:34]: Good comment. I also will create the title as a component.

**Review by AreYouIvan — APPROVED** [review]
Good job! :D
Pending finding what causes the error in deployment.

**Pull request merged**